amoro-core/src/main/java/org/apache/amoro/table/TableMetaStore.java

```diff
@@ -541,6 +541,8 @@ public static class Builder {
     private byte[] krbKeyTab;
     private byte[] krbConf;
     private String krbPrincipal;
+    private String accessKey;
+    private String secretKey;
```

Comment:
It doesn't look like these two values ​​are read anywhere

Reply:
hi @baiyangtx, we need to read ak/sk in PaimonCatalogFactory when using s3 based catalog.